cuda/test/matrix/ell_kernels.cpp

```diff
@@ -160,6 +248,39 @@ TEST_F(Ell, SimpleApplyWithStrideIsEquivalentToRef)
 }
 
 
+TEST_F(Ell, MixedSimpleApplyWithStrideIsEquivalentToRef1)
+{
+    set_up_apply_data(532, 231, 1, 300, 600);
```

Comment:
I guess you can extract all these sizes to variables and unify them across tests ?